modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerLogicalTopologyEventsTest.java

```diff
@@ -45,6 +48,9 @@ void testMetaStorageKeysInitializedOnStartWhenTopVerEmpty() throws Exception {
 
         distributionZoneManager.start();
 
+        verify(keyValueStorage, timeout(1000).times(1))
```

Comment:
why do we need changed in this class?

Reply:
It was here before the topologyVersioneDataNodes logic https://github.com/apache/ignite-3/pull/1729/files#diff-eee737dc0330b6a383f01df9ee1d1ae3135655204bdb1a261ff874674d4a6a78L190 - I just thought, that the author of this test earlier wanted to check that the call of keyValueStorage occuried during init.

Reply:
Sorry, wrong test, I think I put it here by mistake, will remove it